class.js

```diff
@@ -1,20 +1,23 @@
 class Foo
 {
-color='negro';peso   =    34;
+  color = 'negro';
+  peso = 34;
+  
   ver()
   {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
-}
-*/
-}
-   }    class Bar   {
-  miMetodo(){if(this.imprimir){console.info('Imprimir!');}}
+    console.info(`Color: ${this.color}. Peso: ${this.peso}`);
+  }
+}    
+
+class Bar   
+{
+  miMetodo()
+  {
+    if(this.imprimir)
```

Comment:
Es recomendable el uso de espacio luego del `if`.
Respetar el formato del código en todos los casos.
```suggestion
    if (this.imprimir)
```